jumanji/training/configs/env/tetris.yaml

```
@@ -0,0 +1,26 @@
+name: tetris
+registered_version: Tetris-v0
+
+network:
+    conv_num_channels: 32
+    tetromino_layers: 16
+    final_layer_dims: [64, 40]
+
+training:
+    num_epochs: 1000
+    num_learner_steps_per_epoch: 10
```

We can reduce the frequency at which the agent is evaluated.
```suggestion
    num_learner_steps_per_epoch: 100
```